File: nodes/examples/excite/excite.js
MNode = require('libmango')

function Excite(){
    this.node = new MNode();
    this.node.iface.add_interface('./excite.yaml',{'excite':this.excite});
    this.node.run();
}

Excite.prototype.excite = function(header,args){
    return ["excited",{'str':args['str']+'!'}]
}

new Excite();

File: nodes/examples/excite/excite.py
from libmango import *

class excite(m_node):
    def __init__(self):
        super().__init__(debug=True)
        self.interface.add_interface('excite.yaml',{'excite':self.excite})
        self.debug_print("running")
        self.run()
    def excite(self,header,args):
        return "excited",{'str':args['str']+'!'}
    
excite()
